src/mbgl/renderer/layers/render_raster_layer.hpp

```diff
@@ -16,11 +18,12 @@ class RenderRasterLayer final : public RenderLayer {
     void evaluate(const PropertyEvaluationParameters&) override;
     bool hasTransition() const override;
     bool hasCrossfade() const override;
-
-    void render(PaintParameters&, RenderSource*) override;
+    void prepare(const LayerPrepareParameters&) override;
+    void render(PaintParameters&) override;
 
     // Paint properties
     style::RasterPaintProperties::Unevaluated unevaluated;
+    std::unique_ptr<ImageLayerRenderData> imageData;
```

Comment:
private

Reply:
It is private in `RenderRasterLayer`

Reply:
:+1: true, was in collapsed block. thanks